sql/core/src/main/scala/org/apache/spark/sql/execution/command/CommandUtils.scala

```diff
@@ -140,11 +140,11 @@ object CommandUtils extends Logging {
   def calculateTotalLocationSize(
```

Comment:
the name should be `calculateLocationsSize`

Reply:
rename to `calculateMultipleLocationSizes` after discuss with wenchen offline.